### DISCUSSION TITLE
[5.X] Is removing Title fields possible?

### DISCUSSION THREAD

**Discussion by boldpaul** [craftcms/cms#15714] [General]
Is it possible to completely remove the title from a field?
In the first screen shot below, i hide the scrolling image title. So in theory, it should not show up when im making an entry to my channel. However, in the second screenshot. The text box from the title field still shows up. This is confusing for someone who isn't necessarily tech savvy. 

Is there anyway to completely remove this title & text box when needed?

![Screenshot 2024-09-10 at 12 54 06 PM](https://github.com/user-attachments/assets/64af6ee5-f24b-4f7a-830b-d5c03c071cb9)

![Screenshot 2024-09-10 at 12 55 01 PM](https://github.com/user-attachments/assets/47a353a0-dc14-4a59-8a46-8db0087baacf)

**Comment by boldpaul** [comment]
Found out you can just rename the title and use it for something else, such as a alt tag text. Guess i answered my own question.

**Comment by brandonkelly** [comment]
That “Hide” setting is only for hiding the _label_ of the Title field.

If you want to hide the Title field completely, toggle the “Show the Title field” setting on the entry type.
